matchtree.go

```diff
@@ -181,6 +181,10 @@ type branchQueryMatchTree struct {
 	docID     uint32
 }
 
+func (t *branchQueryMatchTree) branchMask() uint64 {
+	return t.fileMasks[t.docID]
+}
```

Comment:
I think branchMask is misleading since you have to & it with t.masks? Why not just do that here then the other call sites don't have to interact with t.masks?

```suggestion
func (t *branchQueryMatchTree) branchMask() uint64 {
	return t.fileMasks[t.docID]&t.masks[t.repos[t.docID]]
}
```

Reply:
resolved in https://github.com/sourcegraph/zoekt/pull/565